src/KOKKOS/fix_wall_lj93_kokkos.cpp

```diff
@@ -29,11 +40,156 @@ FixWallLJ93Kokkos<DeviceType>::FixWallLJ93Kokkos(LAMMPS *lmp, int narg, char **a
   kokkosable = 1;
   atomKK = (AtomKokkos *) atom;
   execution_space = ExecutionSpaceFromDevice<DeviceType>::space;
-  datamask_read = EMPTY_MASK;
-  datamask_modify = EMPTY_MASK;
-  virial_global_flag = virial_peratom_flag = 0;
+  datamask_read = X_MASK | V_MASK | MASK_MASK;
```

Comment:
Same comment as above.

Reply:
there is only one parallel region in FixWallLJ93Kokkos, then it is fine to set datamask_read. no change needed here.